src/actions/index.ts

```diff
@@ -48,3 +43,16 @@ export const fetchUserFromDB = (username, password) => {
       .catch(error => error.message);
   };
 };
+
+export const createUserInDB = (username, email, password) => {
```

Comment:
remember to type both the arguments `(username: string, email: string, password: string)` and what the function returns. In this case, something like `(arguments): (dispatch) => void` or something.